lib/utils/compose-types.d.ts

```diff
@@ -80,9 +80,7 @@ export interface ComposeProject {
 }
 
 export interface Release {
-	client: ReturnType<
-		typeof import('@balena/compose/dist/release').createClient
-	>;
+	client: import('@balena/compose/dist/release').Request['client'];
```

Comment:
@thgreasi wouldn't `typeof import('@balena/compose').release['client']` work? I have a local PR I am working to remove all these deeps imports. If we expose `relçease/models` on balena-compose I guess they all can be gone

Reply:
Nice spot. I wouldn't block though on exposing the rest tbh.